src/Base/BaseTax.php

```diff
@@ -95,6 +95,13 @@ public function init()
             'rewrite' => $rewrite,
         );
 
+        switch ($post_type) {
```

Comment:
Do we need the switch case?
Can we pass multiple args to the filter and do the check on the theme level or when needed?

Reply:
regarding the switch, currently we might not need it since we have only one taxonomy, but in case another taxonomy added later it will be used.